coresimd/src/x86/i686/sse2.rs

```diff
@@ -111,17 +184,53 @@ extern "C" {
 mod tests {
     use stdsimd_test::simd_test;
 
+    use core::mem;
     use v128::*;
-    use v64::i32x2;
+    use v64::{__m64, i32x2, u32x2};
     use x86::i686::sse2;
 
+    #[simd_test = "sse2"]
+    unsafe fn _mm_add_si64() {
```

Comment:
It looks like this and the two tests below it are causing the failures on Windows, could a `#[cfg]` be added with a FIXME perhaps?